src/CoreWCF.Http/tests/Helpers/ServiceHelper.cs

```diff
@@ -198,6 +198,53 @@ public static Binding GetBufferedModHttp1Binding()
             })
             .UseStartup<TStartup>();
 
+        public static IWebHostBuilder CreateHttpsWebHostBuilderWithHttpSys<TStartup>(ITestOutputHelper outputHelper = default) where TStartup : class =>
+            WebHost.CreateDefaultBuilder(Array.Empty<string>())
+#if DEBUG
+            .ConfigureLogging((ILoggingBuilder logging) =>
+            {
+                if(outputHelper != default)
+                    logging.AddProvider(new XunitLoggerProvider(outputHelper));
+                logging.AddFilter("Default", LogLevel.Debug);
+                logging.AddFilter("Microsoft", LogLevel.Debug);
+                logging.SetMinimumLevel(LogLevel.Debug);
+            })
+#endif // DEBUG
+            .UseHttpSys(options =>
+            {
+                options.AllowSynchronousIO = true;
+                options.Authentication.AllowAnonymous = false;
+                options.Authentication.Schemes = Microsoft.AspNetCore.Server.HttpSys.AuthenticationSchemes.Negotiate
+                                                 | Microsoft.AspNetCore.Server.HttpSys.AuthenticationSchemes.NTLM;
+                options.MaxConnections = null;
+                options.MaxRequestBodySize = 30000000;
+                options.UrlPrefixes.Add("https://localhost:5001");
+            })
```

Comment:
You need to use one of the IIS Express preconfigured ports. This is failing in the PR test run because the certificate isn't configured on port 5001. Use a port number between 44300 and 44399 and the SSL cert will be configured correctly.

Reply:
Fixed

Reply:
Looks like SSL is failing after changing to port 44300: https://dev.azure.com/dotnet/CoreWCF/_build/results?buildId=72369&view=logs&j=61b3ff45-772e-578d-ad5f-a6aa4ee834dd&t=73006cae-d2fd-57c9-bf9e-49e14522176c&l=310